kedro/framework/session/session.py

```diff
@@ -192,6 +193,8 @@ def create(  # pylint: disable=too-many-arguments
 
     def _get_logging_config(self) -> Dict[str, Any]:
         logging_config = self._get_config_loader()["logging"]
+        if isinstance(logging_config, omegaconf.DictConfig):
+            logging_config = OmegaConf.to_container(logging_config)
```

Comment:
Is there a reason why logging config only works with it being a primitive container?

Reply:
Yes actually two reasons:
1. https://github.com/kedro-org/kedro/blob/main/kedro/framework/session/session.py#L197-L199 Needs a dict to properly convert the absolute paths
2. this line expects a regular dict: https://github.com/kedro-org/kedro/blob/main/kedro/framework/project/__init__.py#L226